### ISSUE TITLE
iOS: text field expand under keyboard

### ISSUE THREAD

**Issue by lxoo** [etesync/etesync-notes#139]
OS: iOS 14.3
App version: 1.4.1

When editing a note, the text goes beyond the keyboard, thus making it impossible to see what is being typed. Moreover, it makes it impossible to read what’s under the keyboard. The text field is effectively expanding behind the keyboard, instead of adapting its size.

**Comment by zecakeh** [comment]
It has been fixed in version 1.5.1. Can you try to use the `Download Updates` button at the bottom of the Settings ?

We should release a new version on the App Store soon-ish.

**Comment by lxoo** [comment]
Thank you. I do not see that button, I will wait for the new version to be released on the App Store.

**Comment by tasn** [comment]
Yeah, it's not there for iOS. I hope to make a new iOS release soon. It's just such a PITA to deal with Apple. :P
